src/goto-programs/compute_called_functions.cpp

```diff
@@ -54,13 +54,10 @@ void compute_address_taken_functions(
   const goto_programt &goto_program,
   std::unordered_set<irep_idt> &address_taken)
 {
-  forall_goto_program_instructions(it, goto_program)
-  {
-    if(it->has_condition())
-      compute_address_taken_functions(it->get_condition(), address_taken);
-
-    compute_address_taken_functions(it->code, address_taken);
-  }
+  for(const auto &i : goto_program.instructions)
+    i.apply([&address_taken](const exprt &expr) {
```

Comment:
This for loop should have braces because its body is spread over multiple lines.